emmopy/emmopy.py

```diff
@@ -0,0 +1,19 @@
+from ontopy import get_ontology
+from typing import Optional
+
+
+def emmo(inferred: Optional[bool] = True):
```

Comment:
```suggestion
def emmo(inferred: bool = True):
```
Since `Optional[bool]` is equal to `Union[None, bool]`, essentially just adding `None` as an option, for this particular argument it isn't relevant. Unless you want to imply that people can pass `None` to `inferred`?
This would make sense, I think, and then if `inferred` is `None` it should be set to `True` in the function, since this is the default.

Reply:
I was thinking inferred=None as a possible (although not common choice). I can remove it if you think that is just cluttering.